.gitignore

```diff
@@ -3,6 +3,7 @@ spec/fixtures/cassettes
 spec/fixtures/credentials.yml
 coverage/**
 .yardoc/**
+.ruby-version
```

Comment:
This file was not included and we can't include it in a gem, because it needs to be able to work in different versions.